junit.xml

```diff
@@ -0,0 +1,438 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
commit by mistakes?

Reply:
yes, can I add junit.xml to the .gitignore file?

Reply:
i will keep it out of this PR for now because it looks like some docker files are referencing it